guides/HowToDoSerial.md

```diff
@@ -0,0 +1,575 @@
+HowToDoSerial(StreamDevice)
+
+How to use StreamDevice and ASYN to create EPICS device support for a simple serial, GPIB, or network attached device
+=====================================================================================================================
+
+W. Eric Norum <wenorum@lbl.gov>
+
+1 Introduction
+--------------
+
+This tutorial provides step-by-step instructions on how to create EPICS support for a simple serial, GPIB (IEEE-488) or network attached device. The steps are presented in a way that should make it possible to apply them in cookbook fashion to create support for other devices. For comprehensive description of all the details of the I/O system used here, refer to the [asynDriver](http://www.aps.anl.gov/epics/modules/soft/asyn/R4-14/asynDriver.pdf) and [StreamDevice](http://epics.web.psi.ch/software/streamdevice/doc/) documentation.
+
+This document isn't for the absolute newcomer though. You must have EPICS installed on a system somewhere and know how to build and run the example application. In particular you must have the following installed:
+
+*   EPICS R3.14.11 or higher.
+*   EPICS ASYN support module version 4.14 or higher.
+*   EPICS StreamDevice support module version 2.4.1 or higher.
+
+Serial, GPIB and network attached devices can now be treated in much the same way. The EPICS StreamDevice and ASYN support modules can use drivers which communicate with serial devices connected to ports on the IOC or to Ethernet/Serial converters or with GPIB devices connected to local I/O cards or to Ethernet/GPIB converters or to network attached devices using VXI-11 or plain TCP ('telnet') communication protocols.
+
+I based this tutorial on the device support I wrote for a Hewlett Packard E3631A power supply. I chose the E3610A as the basis for this tutorial partly because it was what I found lying around but mostly because it uses SCPI-compliant commands which are common to a wide range of devices. Sections 1 through 4 of this tutorial apply to all SCPI (IEEE-488.2) devices.
+
+2 Create a new device support module
+------------------------------------
+
+The easiest way to do this is with the `makeSupport.pl` script supplied with the EPICS ASYN package.
+
+Here are the commands I ran. To make the command examples a little shorter I used shell variables to hold the paths to my installation of EPICS base and the ASYN support module. You can do this with values appropriate for your site or just type the full path names to the commands.
+
+```
```

Comment:
I think you can put makefile syntax here

Reply:
I found this: https://markdown.land/markdown-code-block
it show as make, but I think makefile should work also as I noticed on other files.